### ISSUE TITLE
Why Adam optimizer lead to a very bad result(3.26%mIOU)?

### ISSUE THREAD

**Issue by ZZClub** [jfzhang95/pytorch-deeplab-xception#149]
I made some experiments(pascal VOC and backbone is ResNet) to figure out how settings affect the result.I got a very bad result when I use Adam optimzer. Can someone give some explanation? And how to choose optimizer? THX. 

  | poly: lr_power | sync-bn | freeze-bn | optimizer | mIOU
-- | -- | -- | -- | -- | --
defalut settings in repo | 0.9 | TRUE | FALSE | SGD | 74.83%
expr1 | 3 | TRUE | FALSE | SGD | 77.93%
expr2 | 3 | FALSE | FALSE | SGD | 74.01%
expr3 | 3 | TRUE | TRUE | SGD | 78.20%
expr4 | 3 | TRUE | FALSE | Adam | 3.26%

**Comment by hubertsgithub** [comment]
Adam usually requires lower learning rate. Try e.g. 1e-04 instead of 1e-02

**Comment by ZZClub** [comment]
I made another two experiments.

  | lr | poly: lr_power | sync-bn | freeze-bn | optimizer | mIOU
-- | -- | -- | -- | -- | -- | --
default settings in repo | 0.07 | 0.9 | TRUE | FALSE | SGD | 74.83%
expr1 | 0.07 | 3 | TRUE | FALSE | SGD | 77.93%
expr2 | 0.07 | 3 | FALSE | FALSE | SGD | 74.01%
expr3 | 0.07 | 3 | TRUE | TRUE | SGD | 78.20%
expr4 | 0.07 | 3 | TRUE | FALSE | Adam | 3.26%
expr5 | 0.0001 | 0.9 | TRUE | FALSE | Adam | 78.88%
expr6 | 0.0001 | 3 | TRUE | FALSE | Adam | 78.34%
